cpanfile

```diff
@@ -50,6 +50,7 @@ requires 'Mojo::Base';
 requires 'Mojo::ByteStream';
 requires 'Mojo::IOLoop';
 requires 'Mojo::JSON';
+requires 'Mojo::RabbitMQ::Client';
```

Comment:
please package this and submit it to devel:languages:perl (cpanspec will do most of the work for you)